tests/storage/outofprocess_test.py

```diff
@@ -49,6 +50,13 @@ def oop_cleanup():
     oop.stop()
 
 
+@pytest.fixture(scope='session')
+def usr_file_512():
+    userstorage.PATHS["file-512"].setup()
+    yield userstorage.PATHS["file-512"].path
+    userstorage.PATHS["file-512"].teardown()
```

Comment:
Checking the READMEs, it states that teardown is rarely required. So if no teardown is required and setup can be assumed, I probably could get rid of the fixture altogether. I'll wait for some feedback though.

Reply:
I think you are confusing the upstream userstorage code with vdsm userstorage code.

This fixture repeat "file-512" 3 times, and cannot be changed to support other types of storage, like "file-4k". I'm not sure we need to test this code with 4k block size, but being able to do this later is important.

The way we use this in other files is:

```
@pytest.fixture(
    scope="module",
    params=[
        userstorage.PATHS["file-512"],
        userstorage.PATHS["file-4k"],
    ],   
    ids=str,
)
def user_storage(request):
    storage = request.param
    if not storage.exists():
        pytest.xfail("{} storage not available".format(storage.name))
    return storage
```

We should use the same fixture here, with one parameter "file-512". We an add more block sizes later if needed by adding one line to the fixture params.

Reply:
Addressed in another PR after splitting commits.